dsc_lib/src/dscresources/command_resource.rs

```diff
@@ -6,10 +6,30 @@ use serde_json::Value;
 use std::{collections::HashMap, process::Command, io::{Write, Read}, process::Stdio};
 use crate::dscerror::DscError;
 use super::{dscresource::get_diff,resource_manifest::{ResourceManifest, InputKind, ReturnKind, SchemaKind}, invoke_result::{GetResult, SetResult, TestResult, ValidateResult, ExportResult}};
-use tracing::{debug, info};
+use tracing::{debug, info, trace};
 
 pub const EXIT_PROCESS_TERMINATED: i32 = 0x102;
 
+
+pub fn print_resource_traces(stderr: &String)
```

Comment:
`print` implies direct output to console, perhaps it should be `log_resource_traces()`

Reply:
Updated.